runtimes/pythonrt/local_runner_test.go

```diff
@@ -188,23 +183,41 @@ func Test_parsePyVersion(t *testing.T) {
 	}
 }
 
-func Test_pyExports(t *testing.T) {
-	skipIfNoPython(t)
+//TODO: What to here
```

Comment:
@tebeka ?

Reply:
That's an old comment, we can delete it here.